src/console.cpp

```diff
@@ -194,50 +203,14 @@ bool GetArgumentInteger(uint32 *value, const char *arg)
 	return arg != endptr;
 }
 
-/**
- * Add an item to an alphabetically sorted list.
- * @param base first item of the list
- * @param item_new the item to add
- */
-template<class T>
-void IConsoleAddSorted(T **base, T *item_new)
-{
-	if (*base == nullptr) {
-		*base = item_new;
-		return;
-	}
-
-	T *item_before = nullptr;
-	T *item = *base;
-	/* The list is alphabetically sorted, insert the new item at the correct location */
-	while (item != nullptr) {
-		if (strcmp(item->name, item_new->name) > 0) break; // insert here
-
-		item_before = item;
-		item = item->next;
-	}
-
-	if (item_before == nullptr) {
-		*base = item_new;
-	} else {
-		item_before->next = item_new;
-	}
-
-	item_new->next = item;
-}
-
 /**
  * Remove underscores from a string; the string will be modified!
  * @param[in,out] name String to remove the underscores from.
  * @return \a name, with its contents modified.
  */
-char *RemoveUnderscores(char *name)
+static std::string &RemoveUnderscores(std::string &name)
 {
-	char *q = name;
-	for (const char *p = name; *p != '\0'; p++) {
-		if (*p != '_') *q++ = *p;
-	}
-	*q = '\0';
+	name.erase(std::remove_if(name.begin(), name.end(), [](char c) {return c == '_'; }), name.end());
```

Comment:
Spacing around the "{return" and "; }" is inconsistent. I would expect there to be a space after the { or the space before the } to go.